shared-worker/index.html

```diff
@@ -0,0 +1,6 @@
+<script src="../firebase/firebase-app.js"> </script>
+<script src="../firebase/firebase-firestore.js"> </script>
```

Comment:
I did it as copied files for simplicity's sake, but adding this as a package.json file would give developers the rest of Firebase to play around with if they forked this.

How does using firebase as a node_module work in this case? Does importing the script again override what's in the node_module?

Reply:
Oh, you can probably ignore me.  I was thinking they would run `npm install` which will install the firebase module into node_modules/ and then this html page will reference it from there...  but I forgot we're using an experimental build so we explicitly don't want to use the real firebase module...  so that doesn't work out.